tor/core/user_interaction.py

```diff
@@ -71,8 +72,10 @@ def process_coc(post, config):
     # instead. If they're actually new, then send a message to slack.
     if result == 1:
         send_to_modchat(
-            f'<http://www.reddit.com/u/{post.author.name}|u/{post.author.name}>'
-            f' has just accepted the CoC! {modchat_emote}',
+            f'<{reddit_url.format(\'/user/\' + post.author.name)}|u/{post.author.name}>'
```

Comment:
what's going on here?

Reply:
Just needs to be `reddit_url.format('/user/' + post.author.name)`. No escaping necessary :)

Reply:
Haha never seen highlighting red like that. Removed the escaping.

Reply:
Not quite, because then the string literal would be ended (since it already uses single quotes). The best option is to change to double quotes in the fstring expression, ie. `reddit_url.format("/user/" + post.author.name)`.